### ISSUE TITLE
No inputs were found in config file [typescript]

### ISSUE THREAD

**Issue by jerroydmoore** [DamonOehlman/detect-browser#95]
When running in Visual Code, The following error is reported: 

```
No inputs were found in config file '***/node_modules/detect-browser/tsconfig.json'.
Specified 'include' paths were '["./src/**/*.ts"]' and 'exclude' paths were '["node_modules"]'.
```

According to [SO](https://stackoverflow.com/questions/41211566/tsconfig-json-buildno-inputs-were-found-in-config-file), it's because there are no ts files in the project. It recommends adding an empty ts file. But maybe a better option is to remove the tsconfig.json until the project starts to add ts files?

**Comment by DamonOehlman** [comment]
@jerroydmoore Apologies for the delay in getting back to you.  I think I understand why this is happening and that's because during the npm publish step we remove all the `src/*` files.  I expect that if I add `tsconfig.json` to `.npmignore` that should fix the problem.  Will do a little bit of reading on that and if it does seem to be the appropriate fix then I'll do that.

**Comment by DamonOehlman** [comment]
Yep, simple fix.  Published in `4.2.1`.  I'll leave the issue open and if when you get a chance if you can validate that the issue has been resolved that would be appreciated.
